Sources/MainScene/ViewControllers/TimeSettingViewController.swift

```diff
@@ -53,7 +53,7 @@ final class TimeSettingViewController: UIViewController {
     }
 
     private lazy var closeButton = UIButton().then {
-        $0.setImage(UIImage(systemName: "xmark"), for: .normal)
+        $0.setImage(UIImage(named: "closeButton"), for: .normal)
```

Comment:
|AS-IS | TO-BE|
|:-:|:-:|
|<img width="380" alt="스크린샷 2024-07-08 오전 1 41 55" src="https://github.com/HGU-iOS-Study-Group/Pomodoro/assets/46087477/6202fd31-2b92-4cd6-a326-21e6a3ceba79">|<img width="385" alt="스크린샷 2024-07-08 오전 1 41 26" src="https://github.com/HGU-iOS-Study-Group/Pomodoro/assets/46087477/2b752df9-4797-413d-8679-c38d6fcc1adf">|